### PR TITLE
EVG-6478: hide authentication related UI

### PR DESCRIPTION
**Pull request by dominoweir**
This set of changes includes hiding login related UI components (the code is still there, just commented out), and adding the username as a URL parameter when viewing patches.

### CODE REVIEW

File: src/components/app/App.tsx [hunk 1]
```diff
@@ -49,7 +49,7 @@ export class Evergreen extends React.Component<Props, State> {
   constructor(props: Props) {
     super(props);
     this.state = {
-      APIClient: rest.EvergreenClient("", "", "", "", true),
+      APIClient: rest.EvergreenClient("https://evergreen.mongodb.com/api", "https://evergreen.mongodb.com")
```
Comment (john-m-liu) [inline src/components/app/App.tsx]: This should come from the config

Comment (dominoweir) [reply src/components/app/App.tsx]: You're right- I misunderstood this ticket initially and thought all the config stuff should be gone too. I've fixed this now.

**Comment by dominoweir** [comment]
I have now removed that menu. The only thing with that is now I don't have a way to upload a config file for testing purposes, although I don't know how much that matters at this point.

File: src/components/patch/PatchContainer.tsx [hunk 2]
```diff
@@ -67,6 +67,14 @@ export class PatchContainer extends React.Component<Props, State> {
 
   private loadPatches = () => {
     if (this.state.hasMore && !this.state.isSearching) {
+      let username = this.props.username;
+      const urlParams = window.location.href.split("?");
```
Comment (john-m-liu) [inline src/components/patch/PatchContainer.tsx]: I would use URLSearchParams to parse this: https://davidwalsh.name/query-string-javascript

**Pull request merged**